site/src/docs/components/header/code.mdx

```diff
@@ -308,12 +398,12 @@ import { Header, IconSearch } from 'hds-react';
 
 ### Properties
 
-| Property                    | Description                       | Values         | Default value |
-| --------------------------- | --------------------------------- | ---------------| ------------- |
-| `ariaLabel`                 | Aria-label for describing header. | `string`       | -             |
-| `className`                 | Additional class names to apply.  | `string`       | -             |
-| `id`                        | Id for the element.               | `string`       | -             |
-| `theme`                     | Header theme.                     | `light`,`dark` | `light`       |
+| Property                    | Description                       | Values                              | Default value |
+| --------------------------- | --------------------------------- | ------------------------------------| ------------- |
+| `ariaLabel`                 | Aria-label for describing header. | `string`                            | -             |
+| `className`                 | Additional class names to apply.  | `string`                            | -             |
+| `id`                        | Id for the element.               | `string`                            | -             |
+| `theme`                     | Header theme.                     | `light`,`dark`, `HeaderTheme`       | `light`       |
```

Comment:
I think we could actually list all three `light, dark, HeaderCustomTheme`, `HeaderTheme` being the union type might be confusing for the user 🤔